### PR TITLE
slight tidy

### PR DESCRIPTION
**Pull request by casperdcl**
after #109 :)

### CODE REVIEW

File: eqt/ui/UIFormWidget.py [hunk 1]
```diff
@@ -488,10 +471,10 @@ def __init__(self, parent=None):
 
 
 class FormDockWidget(QtWidgets.QDockWidget):
-    def __init__(self, parent=None, title=None):
+    def __init__(self, parent=None, title=''):
```
Comment (paskino) [inline eqt/ui/UIFormWidget.py:474]: why this default and accept `title=None`?

Comment (casperdcl) [reply eqt/ui/UIFormWidget.py:474]: honestly I'd prefer removing the lines below:

```python
        if title is None:
            title = ''
```

but I left them in just for backwards compatibility.

Comment (DanicaSTFC) [reply eqt/ui/UIFormWidget.py:474]: > honestly I'd prefer removing the lines below:
> 
> ```python
>         if title is None:
>             title = ''
> ```
> 
> but I left them in just for backwards compatibility.

we added these lines in https://github.com/TomographicImaging/eqt/pull/102/files
to fix some problems. the default title = None was already set then.

File: eqt/ui/UIFormWidget.py [hunk 2]
```diff
@@ -392,18 +378,15 @@ def applyWidgetState(self, name, state, role=None):
...
-            if role in ['label', 'field']:
+            if role in ('label', 'field'):
                 name_role = name + '_' + role
             else:
                 raise ValueError(f'Role must be either "label", "field" or None. Got {role}.')
         else:
             name_role = f'{name}_field'
 
         # retrieve widget
-        try:
```
Comment (DanicaSTFC) [inline eqt/ui/UIFormWidget.py:403]: Why are we removing this KeyError?

**Review by DanicaSTFC — CHANGES_REQUESTED** [review]

**Review by DanicaSTFC — APPROVED** [review]

**Comment by DanicaSTFC** [comment]
I fixed the problems and approved the changes.

**Pull request merged**